snuba/subscriptions/subscription.py

```diff
@@ -33,7 +33,8 @@ def create(self, data: SubscriptionData, timer: Timer) -> SubscriptionIdentifier
         # If there is a delegate subscription, we need to run both the SnQL and Legacy validator
         if isinstance(data, DelegateSubscriptionData):
             self._test_request(data.to_snql(), timer)
-            self._test_request(data.to_legacy(), timer)
+            # Commented this out because `LegacySubscriptionData` is deprecated
+            # self._test_request(data.to_legacy(), timer)
```

Comment:
What fails if you leave this in? `LegacySubscriptionData` is deprecated for us (Sentry) but not for OSS or Onprem users. If we remove this now we might let those customers put their alerts into a bad state.

Reply:
It fails if I am trying to build a sessions query without an `organization` param